README.rst

```diff
@@ -9,7 +9,7 @@ Installation instructions
 
 Create a conda environment for labadmin::
 
-   conda create -n labadmin psycopg2 python=2 click requests
+   conda create -n labadmin psycopg2 python=2 click requests bioconda java-jdk
```

Comment:
@sjanssen2 apparently you didn't had a problem installing this with the current instructions, right? If so, let's move forward with this and not block this PR. On the other hand, if you have installed this using conda, simply post your commands and @antgonza will update the PR with your commands.